src/components/Search/SearchDropdown.tsx

```diff
@@ -20,6 +20,7 @@ interface SearchDropdownProps {
   searchType: InputType;
   repository: Repository['name'];
   labelIdInfo: string;
+  isOverTimeBase: boolean;
```

Comment:
Suggestion, use an optional string prop instead:

```suggestion
  name?: string;
```

Then you don't need to specify it when not needed, and you can use it directly below.

Reply:
Right! Much better. Thanks